processor/src/operations/u32_ops.rs

```diff
@@ -453,4 +458,12 @@ mod tests {
         }
         expected
     }
+
+    fn build_expected_helper_registers(values: &[u32]) -> [Felt; 6] {
```

Comment:
Should we use `NUM_USER_OP_HELPERS` here instead of hard-coding `6`?

Reply:
I followed what we had in the above method `build_expect`, where the length has been explicitly defined. 

I've updated both in the [4a59978](https://github.com/maticnetwork/miden/pull/543/commits/4a599787fa1a0e9bbe13e40db09a5ea145b242e7).